.gitpod.yml

```diff
@@ -11,7 +11,7 @@ tasks:
     openMode: tab-after
     command: |
       go generate -v ./...
-      echo "Use example.sh to build and test the example"
```

Comment:
Interesting.

I removed that because it takes a long time to run *every* time the ephemeral env is (re)started.
I feel it is a stand in for integration tests but def curious on your use case for running it every time?

Reply:
I added it back in because it shows a user who's unfamiliar with dazzle what dazzle does. Things happen when you open a workspace. Also, it can always be canceled at any point.